Adyen/UI/Form/Items/IssuerList/FormIssuerListItem.swift

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Describes a phone number entry form item.
+@_spi(AdyenInternal)
+public final class FormIssuerListItem: FormTextItem {
```

Comment:
I believe this class/file is not needed, correct me if I am wrong.

Reply:
Yes, I have removed them. Thanks for pointing it out :)